xc/common/cmake/arch_define.cmake

```diff
@@ -106,7 +106,7 @@ function(ADD_XC_ARCH_DEFINE)
     CELLS_SIM ${YOSYS_CELLS_SIM} ${VPR_CELLS_SIM}
     VPR_ARCH_ARGS ${VPR_ARCH_ARGS}
     RR_PATCH_TOOL
-      ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/prjxray_routing_import.py
+      ${symbiflow-arch-defs_SOURCE_DIR}/xc/common/utils/${PRJRAY_NAME}_routing_import.py
```

Comment:
I'd prefer if don't hard fork these scripts.  They aren't small pieces of code to begin with, and I'm not excited about needing to maintain two sets.  Can we factor out the xc7 vs xcu specific stuff?

Reply:
Actually I agree. I took that approach with `assign_tile_pin_direction.py`.

So the idea would be to have the common code in eg. `rounting_import.py` plus two wrapper scripts `prjxray_routing_import.py` and `prjuray_routing_import.py` that would contain the specific code. These specific scripts would then be used in CMake. Does that make sense?

Reply:
Sounds good to me.